docs/src/gallery/tutorials/run_ensemblelda.py

```diff
@@ -0,0 +1,155 @@
+r"""
+Ensemble LDA
+============
+
+Introduces Gensim's EnsembleLda model
+
+"""
+
+import logging
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)
+
+###############################################################################
+# This tutorial will explain how to use the EnsembleLDA model class.
+#
+# EnsembleLda is a method of finding and generating stable topics from the results of multiple topic models,
+# it can be used to remove topics from your results that are noise and are not reproducible.
+#
+
+###############################################################################
+# Corpus
+# ------
+# We will use the gensim downloader api to get a small corpus for training our ensemble.
+#
+# The preprocessing is similar to :ref:`sphx_glr_auto_examples_tutorials_run_word2vec.py`,
+# so it won't be explained again in detail.
+#
+
+import gensim.downloader as api
+from gensim.corpora import Dictionary
+from nltk.stem.wordnet import WordNetLemmatizer
+
+lemmatizer = WordNetLemmatizer()
+docs = api.load('text8')
+docs = [[lemmatizer.lemmatize(token) for token in doc] for doc in docs]
+dictionary = Dictionary(docs)
+dictionary.filter_extremes(no_below=20, no_above=0.5)
+corpus = [dictionary.doc2bow(doc) for doc in docs]
```

Comment:
Is it a good idea to keep the whole corpus in a list (in RAM) like this?

I know it makes the tutorial code simpler, but people tend to copy-paste… Maybe at least add a note that document streaming is available / preferred, for real-world use?

Reply:
I went for the streaming option instead of adding a note.